lib/components/notice/notice.less

```diff
@@ -87,6 +74,8 @@
                 --_no-code-fc: var(--white);
             }
         }
+
+        --_no-bc: var(--black-400);
```

Comment:
This applies `--_no-bc: var(--black-400);` to the base high contrast style which can then be overridden by the `.highcontrast-mode` style rules defined in `.generate-variant-variables` (line 20).